imports/local/imports_local.go

```diff
@@ -18,7 +18,7 @@ import (
 	// _ "github.com/emccode/libstorage/drivers/storage/mock/client"
 	//_ "github.com/emccode/libstorage/drivers/storage/openstack/client"
 	//_ "github.com/emccode/libstorage/drivers/storage/rackspace/client"
-	//_ "github.com/emccode/libstorage/drivers/storage/scaleio/client"
+	_ "github.com/emccode/libstorage/drivers/storage/scaleio"
```

Comment:
Hi @clintonskitson,

This actually imports the entire SIO package into the libStorage client. We probably don't want that.